app-misc/opentrack/opentrack-9999.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop flag-o-matic toolchain-funcs
+
+DESCRIPTION="Head tracking software for MS Windows, Linux, and Apple OSX"
+HOMEPAGE="https://github.com/opentrack/opentrack"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/opentrack/opentrack.git"
+else
+	SRC_URI="https://github.com/opentrack/opentrack/archive/refs/tags/${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/opentrack-${P}"
+fi
+
+LICENSE="ISC"
+SLOT="0"
+
+IUSE="neuralnet wine opencv openmp"
+
+REQUIRED_USE="neuralnet? ( openmp opencv )"
+
+DEPEND="
+	neuralnet? ( sci-libs/onnxruntime-bin )
+	opencv? ( media-libs/opencv:= )
+	wine? ( virtual/wine )
+	dev-libs/libevdev
+	dev-libs/libusb:1
+	dev-qt/qtbase:6
+	sys-process/procps:=
+	x11-libs/libX11
+"
+RDEPEND="${DEPEND}"
+
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+pkg_setup() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+src_prepare() {
+	# work around hard-coded docs path
+	sed -e 's#share/doc/opentrack#share/doc/'${P}'#g' \
+		-i cmake/*.cmake || die
+
+	cmake_src_prepare
+}
+
+src_configure() {
+	filter-lto
```

Comment:
Did you try this with lto?  What happens?

Reply:
The application compiles with LTO but it renders graphical artifacts (red borders around widgets) and crashes on almost any click on a button. Reported here: https://github.com/opentrack/opentrack/issues/2009

Reply:
Thanks for the heads up: Works with the Qt6 build now, also reported upstream.